automation/previous-hpp.sh

```diff
@@ -18,7 +18,7 @@
 #
 
 set -ex
-export TARGET=k8s-1.22
+export TARGET=k8s-1.23
```

Comment:
you are fine with it as 1.23 and not 1.24 ?
why is it different ?

Reply:
We test both 1.23 and 1.24 so we have a 'latest' and 'previous' lane. The previous lane is now 1.23 and latest is 1.24.